frb_codegen/src/library/codegen/generator/wire/rust/spec_generator/misc/wire_func.rs

```diff
@@ -238,11 +242,15 @@ fn generate_code_closure(
     code_call_inner_func_result: &str,
 ) -> String {
     let codec = (func.codec_mode_pack.rust2dart.to_string()).to_case(Case::Snake);
+    let mut maybe_result = "".to_string();
+    if matches!(&func.output, IrType::RustAutoOpaque(_)) && func.fallible() {
+        maybe_result = "-> Result::<_,anyhow::Error>".to_string()
+    };
```

Comment:
nit: what about if-else, s.t. it can be non-mut

let maybe_result = if ... { "-> Result..." } else {""};